instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/__init__.py

```diff
@@ -252,6 +249,11 @@ def _get_operation_name(handler, request):
     return f"{class_name}.{request.method.lower()}"
 
 
+def _get_full_handler_name(handler):
+    klass = type(handler)
+    return f"{klass.__module__}.{klass.__qualname__}"
```

Comment:
would this work fine for dynamically generated handlers within a function? I know that is not very common but sounds like an edge case. Any concerns regarding that?

Reply:
I believe `__qualname__` would do the right thing with closures, see the [examples in PEP 3155](https://www.python.org/dev/peps/pep-3155/#example-with-nested-functions)